From: Xuanye Liu <liuqiye2025@163.com>
Subject: mm: remove outdated filename comment in percpu-stats.c
Date: Wed, 2 Jul 2025 14:23:59 +0800

The comment had the old filename. It's also unnecessary, so drop it.

Link: https://lkml.kernel.org/r/20250702062400.207619-1-liuqiye2025@163.com
Signed-off-by: Xuanye Liu <liuqiye2025@163.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/percpu-stats.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/percpu-stats.c~mm-remove-outdated-filename-comment-in-percpu-statsc
+++ a/mm/percpu-stats.c
@@ -1,6 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0-only
 /*
- * mm/percpu-debug.c
  *
  * Copyright (C) 2017		Facebook Inc.
  * Copyright (C) 2017		Dennis Zhou <dennis@kernel.org>
_
